### PR TITLE
add confirmation modal for text & multimedia additions

### PR DESCRIPTION
**Pull request by MilanBarande**
Add a confirmation modal in the landing page administration when a user wants to add a Text & Multimedia section

### CODE REVIEW

File: assembl/static2/js/app/components/administration/landingPage/manageModules.jsx [hunk 1]
```diff
@@ -36,33 +35,55 @@ export const DumbManageModules = ({
...
-        <Translate value="administration.landingPage.manageModules.helper" />
-      </p>
-      <div className="two-columns-admin">
-        <div className="column-left">
-          <SelectModulesForm lang={locale} moduleTypes={moduleTypes} modulesById={modulesById} toggleModule={toggleModule} />
-          <div className="margin-xl">
-            <Button className="button-submit button-dark" onClick={() => createModule(enabledModules.size - 2)}>
-              <Translate value="administration.landingPage.manageModules.textAndMultimediaBtn" />
-            </Button>
+}: Props) => {
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/components/administration/landingPage/manageModules.jsx]: Recently, after speaking with @pascalhuynh regarding making components easier to create/mock. As is, this component's `submitButton` is baked in and cannot be changed. So perhaps you could pull the `displayConfirmationModal` out of the definition of the component, and pass it in as a prop, like `handleSubmitButton`. And the `displayConfirmationModal` method can take the component props as input and pick up what it needs internally. It's a better way to mock/test these components in future.

File: assembl/static2/js/app/utils/translations.js [hunk 2]
```diff
@@ -707,7 +707,8 @@ const Translations = {
           title: "Administrer les modules",
           helper: "Choisissez les modules que vous souhaitez voir apparaître dans la page d'accueil et leurs emplacements.",
           textAndMultimedia: "Texte & Multi-média",
-          textAndMultimediaBtn: "Ajouter un module Texte & Multi-media"
+          textAndMultimediaBtn: "Ajouter un module Texte & Multi-media",
+          confirmationModal: "Voulez vous ajouter un module Texte & Multimédia supplémentaire à la landing page ?"
```
Comment (vincentfretin) [inline assembl/static2/js/app/utils/translations.js]: `Voulez-vous` avec un tiret. :) https://leconjugueur.lefigaro.fr/frtraitunionquestion.php

File: assembl/static2/js/app/components/administration/landingPage/manageModules.jsx (hunk 1 above)
Comment (amensouissi) [reply assembl/static2/js/app/components/administration/landingPage/manageModules.jsx]: I think that it's a specific business component and not abstract (ui or business component). We don't need to reuse it. in my opinion the solution is sufficient.

**Comment by vincentfretin** [comment]
you can merge

**Pull request merged**